### PR TITLE
Add CommandBlock functionality

### PR DESCRIPTION
**Pull request by piepie62**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ evaluationDependsOn('Mixin')
 evaluationDependsOn('SpongeAPI')
 
 // Default tasks
-defaultTasks 'build', 'licenseFormat'
+defaultTasks 'build'
```
Comment (mproncace) [inline build.gradle]: Why did you disable the license task?

File: build.gradle [hunk 2]
```diff
@@ -176,15 +169,6 @@ jar {
...
 
-javadoc {
-    source = [
-        project("Mixin").sourceSets.main.java,
-        project("SpongeAPI").sourceSets.main.java,
-        sourceSets.main.java,
-        "src/example/java"
-        ]
-}
-
```
Comment (mproncace) [inline build.gradle]: Why is this removed?

File: src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java [hunk 3]
```diff
@@ -32,4 +34,16 @@ public SpongeHorseColor(int color, String name) {
...
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public void serialize(DataSource source) {
+		// TODO Auto-generated method stub
+		
+	}
+
```
Comment (mproncace) [inline src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java]: This doesn't look like it relates the PR. I'd suggest removing it.

File: src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java [hunk 4]
```diff
@@ -104,8 +104,7 @@ public BlockLoc getBlock(int x, int y, int z) {
         return new BlockWrapper(this, x, y, z);
     }
 
-    @Override
-    public Biome getBiome(Vector3d position) {
+    public BiomeType getBiome(Vector3d position) {
```
Comment (mproncace) [inline src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java]: This should probably be fixed by the core team along with other minor bugs rather than in this PR.

**Comment by simon816** [comment]
Your branch is not in sync with master, try a rebase

File: src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java (hunk 3 above)
Comment (piepie62) [inline src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java]: For some reason, it showed a lot of errors.

**Comment by piepie62** [comment]
I am closing this because I need to add some major changes to this.
